src/js/flick.js

```diff
@@ -11,25 +12,19 @@ $.flick.inputs = {
   tilt: require('./inputs/tilt')
 };
 
-$.flick.defaults = {
-  input: 'tilt' // String or function
-};
-
 /**
  * Make a flicky thing!
  *
  * @param {object} options Object containing some options.
  */
 $.fn.flick = function flick(options) {
-  options = $.extend({}, $.flick.defaults, options);
```

Comment:
This line should be kept in. Have the default `input` be "auto", and set it to "mouseX" or "tilt" if it's "auto". Lot DRYer and more extensible :)